pagopa/io-app
# chore: [IOPID-517] Fix what's new persistence

## ⚠️ Depends on 4823 ⚠️ ##

## Short description
This PR fixes the what's new state persistence.

## How to test
- Login into the app with FL enable. You should see the what's new bottom sheet.
- Logout from the app.
- Close and reopen the app.
- Login into the app again. You should see the what's new bottom sheet.
